docs/sphinx/developers/logging.txt

```diff
@@ -0,0 +1,51 @@
+Logging
+=======
+
+Logging is the process of writing log messages during execution.
```

Comment:
This is tautological and not very informative for a software developer. I'd remove lines 3-7 altogether and start directly with specific info about logging in BF